src/components/dialogs/two-windings-transformer/two-windings-transformer-creation-dialog.js

```diff
@@ -725,13 +742,16 @@ const TwoWindingsTransformerCreationDialog = ({
     const handleSave = () => {
         setCreationError();
         let isFormValid = true;
+        let tabWithError = [];
```

Comment:
tabWithError is not plural, and has the exact same name than a style.
Can we change it ? tabsWithError, tabWithErrorList, allTabsWithError ...

Reply:
fixed